mkosi/config.py

```diff
@@ -861,7 +861,10 @@ def config_default_output(namespace: argparse.Namespace) -> str:
 
 
 def config_default_distribution(namespace: argparse.Namespace) -> Distribution:
-    detected = detect_distribution()[0]
+    if d := os.getenv("MKOSI_HOST_DISTRIBUTION"):
+        detected = Distribution(d)
+    else:
+        detected = detect_distribution()[0]
```

Comment:
This sidesteps this here neatly
```suggestion
    if d := os.getenv("MKOSI_HOST_DISTRIBUTION"):
        return Distribution(d)
    
    detected = detect_distribution()[0]
```